app_server/app/models/Storage.js

```diff
@@ -241,9 +242,12 @@ Class.updateParticulars = function(userId, newParticulars) {
  * @return {Promise<List<Sequelize.object>>} - a list of users
  *         {False} on fail
  */
-Class.getListOfUsers = function() {
+Class.getListOfUsers = function(originalFilters) {
+
+  var filters = mapParams(originalFilters);
```

Comment:
I believe this should be fine as the below, assuming I got my JS right?
```
function (filters) {
  filters = mapParams(filters);
````
That variable naming would make more sense from outside the file at least.